### ISSUE TITLE
Conflict in sbt-assembly

### ISSUE THREAD

**Issue by PawelJ-PL** [janino-compiler/janino#95]
Hi,

I'm using sbt-assembly to build jar. But with Janino in the newest version (3.0.13) I' getting following error:

```
error] 1 error was encountered during merge
[error] java.lang.RuntimeException: deduplicate: different file contents found in the following:
[error] ~/.ivy2/cache/org.codehaus.janino/janino/jars/janino-3.0.13.jar:javac.bat
[error] ~/.ivy2/cache/org.codehaus.janino/commons-compiler/jars/commons-compiler-3.0.13.jar:javac.bat
[error] 	at sbtassembly.Assembly$.applyStrategies(Assembly.scala:142)
[error] 	at sbtassembly.Assembly$.x$1$lzycompute$1(Assembly.scala:26)
[error] 	at sbtassembly.Assembly$.x$1$1(Assembly.scala:24)
[error] 	at sbtassembly.Assembly$.stratMapping$lzycompute$1(Assembly.scala:24)
[error] 	at sbtassembly.Assembly$.stratMapping$1(Assembly.scala:24)
[error] 	at sbtassembly.Assembly$.inputs$lzycompute$1(Assembly.scala:68)
[error] 	at sbtassembly.Assembly$.inputs$1(Assembly.scala:58)
[error] 	at sbtassembly.Assembly$.apply(Assembly.scala:85)
[error] 	at sbtassembly.Assembly$.$anonfun$assemblyTask$1(Assembly.scala:249)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:44)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:40)
[error] 	at sbt.std.Transform$$anon$4.work(System.scala:67)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:269)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
[error] 	at sbt.Execute.work(Execute.scala:278)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:269)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error] 	at java.lang.Thread.run(Thread.java:748)
[error] (assembly) deduplicate: different file contents found in the following:
[error] ~/.ivy2/cache/org.codehaus.janino/janino/jars/janino-3.0.13.jar:javac.bat
[error] ~/.ivy2/cache/org.codehaus.janino/commons-compiler/jars/commons-compiler-3.0.13.jar:javac.bat
[error] Total time: 46 s, completed Jul 4, 2019 1:02:45 PM
```

file `javac.bat` exisists in two instances.

There is no such problem with previous version (3.0.12)

**Comment by aunkrig** [comment]
Weird... the following files made it into version 3.0.13:

```
commons-compiler.jar:
File added javac.bat
File added org.codehaus.plexus.compiler.javac.JavacCompiler1999325250648455857arguments
File added org.codehaus.plexus.compiler.javac.JavacCompiler4317238490133791186arguments

janino.jar:
File added javac.bat
File added org.codehaus.plexus.compiler.javac.JavacCompiler3267048155081022619arguments
```

Need to make another release to get rid of these... stay tuned!

**Comment by aunkrig** [comment]
I released version 3.0.14 on July 5 which should fix the problem. Please test.

**Comment by PawelJ-PL** [comment]
I've just tested with 3.0.14 and it works fine. Thank You
